source/frontend/src/features/properties/map/acquisition/ViewSelector.tsx

```diff
@@ -17,6 +17,7 @@ export interface IViewSelectorProps {
   selectedMenuIndex: number;
   setContainerState: (value: Partial<AcquisitionContainerState>) => void;
   onSuccess: () => void;
+  ref: React.RefObject<FormikProps<any>>;
```

Comment:
nit: I think this is not needed here because ViewSelector is created with React.forwardRef(...) that appends that ref to the props, but doesn't hurt to also having it here I think